cmd/cli/cluster.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis"
+)
+
+func renameAnyTagsSubscriptionsKeyForwards(database moira.Database) error {
+	switch d := database.(type) {
+	case *redis.DbConnector:
+		err := d.Client().Rename(d.Context(), "moira-any-tags-subscriptions", "{moira-tag-subscriptions}:moira-any-tags-subscriptions").Err()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func renameAnyTagsSubscriptionsKeyReverse(database moira.Database) error {
+	switch d := database.(type) {
+	case *redis.DbConnector:
+		err := d.Client().Rename(d.Context(), "{moira-tag-subscriptions}:moira-any-tags-subscriptions", "moira-any-tags-subscriptions").Err()
```

Comment:
Rename returns error when key doesn't exist in database. It can be the reason for incorrect exit of migration process.

Reply:
I added a check, thanks.